modules/ossm-tutorial-bookinfo-removing.adoc

```diff
@@ -6,37 +6,55 @@ This TASK module included in the following assemblies:
 [id="ossm-tutorial-bookinfo-removing_{context}"]
 = Removing the Bookinfo application
 
-When you finish with the Bookinfo application, you can remove it by running the cleanup script.
+Follow these steps to remove the Bookinfo application.
+
+.Prerequisites
+
+* {product-title} 4.1 or higher installed.
+* {ProductName} {ProductVersion} installed.
+* Access to the {product-title} Command-line Interface (CLI) also known as `oc`.
+
+[id="ossm-delete-bookinfo-project_{context}"]
+== Delete the bookinfo project
 
-[TIP]
-====
-Several of the other {ProductShortName} tutorials also use the Bookinfo application.  Do not run the cleanup script if you plan to explore other {ProductShortName} tutorials.
-====
 .Procedure
 
-. Download the cleanup script:
-+
+. Log into the {product-title} web console.
 
-----
-$ curl -o cleanup.sh https://raw.githubusercontent.com/Maistra/bookinfo/master/cleanup.sh && chmod +x ./cleanup.sh
-----
+. Click to *Home* -> *Projects*.
 
-. Delete the Bookinfo virtualservice, gateway, and terminate the Pods by running the cleanup script:
-+
+. Click on the `bookinfo` menu {kebab} and choose *Delete Project*.
```

Comment:
Possibly: Click on the `bookinfo` menu {kebab}, and then click *Delete Project*.

Reply:
Fixed.